pytrack/utils/utils.py

```diff
@@ -0,0 +1,34 @@
+"""
+This program and the accompanying materials are made available under the terms of the
+Eclipse Public License v2.0 which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-v20.html
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zincware Project.
+
+Description:
+"""
+
```

Comment:
too much space here?

Reply:
solved in 2175398a95c5813644aef5da0d41bc8ce2df9d21